pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1546,3 +1547,11 @@ func (m *kubeGenericRuntimeManager) ListMetricDescriptors(ctx context.Context) (
 func (m *kubeGenericRuntimeManager) ListPodSandboxMetrics(ctx context.Context) ([]*runtimeapi.PodSandboxMetrics, error) {
 	return m.runtimeService.ListPodSandboxMetrics(ctx)
 }
+
+func removeSensitiveContainerInformation(pod *v1.Pod) *v1.Pod {
```

Comment:
Nit: maybe we should move this to helper.go and add a test for it?

otherwise, LGTM.

Reply:
Sure thing, will do asap

Reply:
done in 9b42e548639